sparrow-py/pysrc/sparrow_py/_timestream.py

```diff
@@ -993,6 +993,44 @@ def cast(self, data_type: pa.DataType) -> Timestream:
         """
         return Timestream(self._ffi_expr.cast(data_type))
 
+    def seconds_since(self, n: Union[Timestream, Literal] = 1) -> Timestream:
+        """
+        Create a Timestream containing the number of seconds since `time`.
+
+        Parameters
+        ----------
+        n : Union[Timestream, Literal]
+            The time to compute the seconds since.
+
+            This can be either a stream of timestamps, a datetime literal, or 
+            an integer literal representing the number of points to lag by. 
```

Comment:
Can we pull this behavior out into a separate function? This is a very subtle difference in usage that produces a substantially different behavior - for example it would be completely reasonable to assume `e.seconds_since(1691761917)` means "the time since a unix timestamp to each value in `e`".

Reply:
Yeah -- I originally had another PR out with a function called `seconds_since_previous` that separated that behavior. I agree with you here. And could add the timestamp functionality to this one.